Improved the design of the loading (#53)
nventive/Uno.Wasm.Bootstrap: Readme.md

The Linker can be disabled completely by setting the `WasmShellILLinkerEnabled` property to 
false. This property has no effect when building with AOT enabled.

### Support for Subresource Integrity
By default, the _msbuild task_ will calculate a hash for binary files in your project and will use the [Subresource Integrity](https://www.w3.org/TR/SRI/)

> to valide you're using the right set of files at runtime.

Reviewer: ```suggestion
to validate that the right set of files are loaded at runtime.
```
